smart-contracts/contracts/RelayPool.sol

```diff
@@ -431,7 +431,6 @@ contract RelayPool is ERC4626, Ownable {
 
   // This function is called externally to claim funds from a bridge.
   // The funds are immediately added to the yieldPool
-  // TODO: handle cases where the origin might have been removed/changed (fees, etc.)
```

Comment:
If the origin was removed, then the user will need to ask the curator to use `processFailedHandler`

Reply:
(curator could also automate that!)

Reply:
And the fees cannot be changed "fast enough" for the solver to ignore the change